src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
-}
+@import "./utils/normalize.scss";
+@import "./utils/variables.scss";
+@import "./utils/functions.scss";
+@import "./utils/mixins.scss";
+
```

Comment:
also we need a reset block=)

Reply:
As I understood it`s not necessary, but I`ll add it, thank you.